src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+
+      <img
+      src="./images/imac.jpeg"
+      alt="iMac27"
+      class="card__image"
+      >
```

Comment:
```suggestion
    <section class="card" data-qa="card">
      <img
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)"
        class="card__image"
      >
```
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md):
3. [CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))
[CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

Reply:
done